tests/test_logger_config.py

```
@@ -0,0 +1,125 @@
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from logger_config import prepare_log_file, setup_file_handler, setup_logging
+
+
+class TestPrepareLogFile:
+    """Tests for prepare_log_file function."""
+
+    def test_prepare_log_file_returns_none_for_none_input(self, tmp_path):
```

Ruff (0.15.2)

[warning] Unused method argument: `tmp_path`

(ARG002)